src/Misc/layoutbin/en-US/strings.json

```diff
@@ -75,6 +76,7 @@
   "EulasSectionHeader": "End User License Agreements",
   "Failed": "Failed: ",
   "FailedDeletingTempDirectory0Message1": "Failed to delete temporary directory '{0}'. {1}",
+  "FailedToAddTags": "Failed to apply tags to agent. Try again or ctrl-c to quit. Alternatively you may go to machine group web page to add tags",
```

Comment:
This message will come when the rest call for UpdateDeploymentMachinesAsync fails, not when tag validation fails.